pkg/operator/ceph/object/controller_test.go

```diff
@@ -615,12 +616,30 @@ func TestCephObjectStoreControllerMultisite(t *testing.T) {
 	objectStore.Spec.Zone.Name = zoneName
 	objectStore.Spec.Gateway.Port = 80
 
-	object := []runtime.Object{
-		objectZone,
-		objectStore,
-		objectZoneGroup,
-		objectRealm,
-		cephCluster,
+	externalObjectStore := &cephv1.CephObjectStore{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      store,
+			Namespace: namespace,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind: "CephObjectStore",
+		},
+		Spec: cephv1.ObjectStoreSpec{},
+	}
+
+	externalObjectStore.Spec.Zone.Name = zoneName
```

Comment:
`Spec.Zone` is not needed for external RGW server

Reply:
@thotz What should be the behavior of an external object store if a user does set this though? Can we validate that Rook does the right thing with it?

Reply:
@BlaineEXE : From code reading, currently `ObjectStore.Spec.Zone` is used by `reconcileMultisiteCRs` which is not called if the cluster is external. And there is no validation check present in `external` store path as well.